frontend/src/app/features/auth/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { addUser, login } from '@store/actions/login-page.actions';
```

Comment:
i'w not import actions just by names - import what you need with alias loginPageActions, so you could access them in code as loginPageActions.addUser. It would be clear in code that you use action - not a local functions called addUser.

Reply:
Fixed